test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1349,7 +1349,7 @@ public void help() throws Exception {
     }
   }
 
-  // @Test(timeout = 45000)
+  @Test
```

Comment:
Was this test previously disabled for a reason?

Reply:
I don't know but it was passing when I ran it.

Reply:
Good enough for me.